ipvlan/bin/openshift-ipvlan-kube-subnet-setup.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -ex
+
+## docker
+if [[ -z "${DOCKER_OPTIONS}" ]]
+then
+    DOCKER_OPTIONS='-b=none --mtu=1450 --selinux-enabled'
```

Comment:
This was copied from ovssubnet/bin/openshift-sdn-kube-subnet-setup.sh and ovssubnet/bin/openshift-sdn-simple-setup-node.sh.  Was the 1450 done to account for overlay network (vxlan) headers?  If so, then I'll remove the MTU bit here since ipvlan doesn't use overlay at all, plus it doesn't use the docker bridge either.